docs/_reference/firestore.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Firestore
+layout: docs
+methods:
+
+  - signature: firestore.getDoc(collectionRef, docRef)
+    id: getDoc
+    generator: true
+    description: Reads the document referred to by this docRef.
+    arguments:
+      - name: collectionRef
+        required: true
+        type: A CollectionReference object.
+      - name: docRef
+        required: true
+        type: A slash-separated path to a document.
+    output: Promise containing non-null firebase.firestore.DocumentSnapshot
```

Comment:
Redux-saga will resolve this promise for us so the output is just a `firebase.firestore.DocumentSnapshot` object here. ;)